app/src/main/java/com/github/giommok/softwaredevproject/FirebaseAccount.java

```diff
@@ -268,7 +278,40 @@ private static void syncGetDiscoveredHeight(DataSnapshot discoveredHeightsFromHa
         for (Map.Entry<String, HashMap<String, String>> entry : entries.entrySet()){
             Object value = entry.getValue();
             long retrievedVal = ((ArrayList<Long>) value).get(0);
-             discoveredPeakHeights.add((int)retrievedVal);
+            discoveredPeakHeights.add((int)retrievedVal);
+        }
+    }
+
+    private static void syncFriendsFromProfile(DataSnapshot parent) {
```

Comment:
I don't remember if we had comments regarding private functions aswell. If we didn't this comment can be discarded

Reply:
Added comment.